src/engines/html5/html5.js

```diff
@@ -930,6 +970,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    */
   _init(source: PKMediaSourceObject, config: Object): void {
     this._config = config;
+    this._source = source;
```

Comment:
need to clean it when the source is being irrelevant

Reply:
when the source is irrelevant? just load media and than it'll be reloaded on init

Reply:
on `CHANGE_SOURCE_STARTED?ENDED` for example